### DISCUSSION TITLE
FR: Mobile SMS based OTP auth system instead of email/password

### DISCUSSION THREAD

**Discussion by arladmin** [directus/directus#19363] [Feature Requests]
Has anyone here been able to implement a workaround/hack using which the users can login into a directus app with mobile SMS based OTP based system instead of email/password? (no passwords creating/saving at all, only realtime SMS OTPs for authentication)

Also, as currently, there isn't any direct existing functionality for this, i would like to mark this as a Feature Request - 
- Auth system using 'mobile number as username' and 'SMS-based OTP as the password'.

Use-case:
- In the Indian market, most of the modern apps now have the default auth system as mobile & sms-OTP, instead of email/password, which is really appreciated (due to increased convenience as well as security) by the Indian users (which are a pretty large population). Lack of such a system in Directus makes it difficult to adopt Directus as the BaaS for apps primarily for the Indian users.

**Comment by rijkvanzanten** [comment]
Closing this as it's not a feature request. Make sure to upvote https://github.com/directus/directus/discussions/17888 and leave a comment about texts!

**Reply by arladmin** [reply]
@rijkvanzanten There's no other category to choose from, when creating a discussion. Does that mean we can now "discuss" only feature requests?

Also, https://github.com/directus/directus/discussions/17888 is mostly regarding logging in using OTP sent via email.
Whereas, my discussion is regarding exclusively using passwordless login via username (mobile number) & SMS (neither email nor passwords).

**Reply by rijkvanzanten** [reply]
> There's no other category to choose from, when creating a discussion. Does that mean we can now "discuss" only feature requests?

We use issues for actionable items and discussions for [feature requests](https://docs.directus.io/contributing/feature-request-process.html#request-a-feature) 👍🏻 

> Also, https://github.com/directus/directus/discussions/17888 is mostly regarding logging in using OTP sent via email.

Yeah, but discussing whether or not that could also be done through text sounds like an appropriate thread to have on that discussion.

> Whereas, my discussion is regarding exclusively using passwordless login via username (mobile number) & SMS (neither email nor passwords).

Using something other than an email for a user's primary authentication identifier sounds like a different feature request to me 🙂

**Comment by arladmin** [comment]
@rijkvanzanten 

> Yeah, but discussing whether or not that could also be done through text sounds like an appropriate thread to have on that discussion.

Cool, will post there as well.


> Using something other than an email for a user's primary authentication identifier sounds like a different feature request to me 🙂

Great! In that case, could we please reopen this discussion (Feature Request)? Thanks.

**Reply by rijkvanzanten** [reply]
> Great! In that case, could we please reopen this discussion (Feature Request)? Thanks.

Please open a new one that follows the RFC template so it's possible to properly discuss the implementation details, thanks!
